lib/katello/engine.rb

```diff
@@ -58,8 +58,12 @@ class Engine < ::Rails::Engine
         },
         :agent => {
           :client_queue_format => 'pulp.agent.%s',
-          :broker_url => 'amqp:ssl:localhost:5671',
-          :event_queue_name => 'katello.agent'
+          :event_queue_name => 'katello.agent',
+          :broker_url => 'amqps://localhost:5671',
+          # puppetize these
+          :broker_ssl_cert_file => '/etc/pki/katello/qpid_router_client.crt',
+          :broker_ssl_key_file => '/etc/pki/katello/qpid_router_client.key',
+          :broker_ssl_ca_file => '/etc/pki/katello/certs/katello-default-ca.crt'
```

Comment:
We do not need to introduce special certificates here for this. We can use the main Foreman client certificates which is a move we are making in general:

https://github.com/theforeman/puppet-foreman/blob/master/templates/settings.yaml.erb#L24-L26